features/step_definitions/viewing_browse_steps.rb

```diff
@@ -78,11 +78,11 @@
 end
 
 Then(/^the A to Z label should be present$/) do
-  assert page.has_content?("A\u200Ato\u200AZ")
+  assert page.has_text?("A\u200Ato\u200AZ")
 end
 
 Then(/^the A to Z label should not be present$/) do
-  assert page.has_no_content?("A\u200Ato\u200AZ")
+  assert page.has_no_text?("A\u200Ato\u200AZ")
```

Comment:
Why is this not just spaces instead of '\u200A' (https://codepoints.net/U+200A?lang=en)

Reply:
We're using a hairspace, i think for design reasons